Cargo.toml

```diff
@@ -15,22 +15,15 @@ futures = "0.3"
 human-panic = "1.0.3"
 tracing = "0.1.26"
 one_err = "0.0.7"
-sodoken = "=0.0.6"
+sodoken = "=0.0.7"
 tokio = { version = "1.11", features = ["full"] }
-# holochain_util = "0.0.13"
-holochain_util = { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-holochain_p2p =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-holochain_types =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-holochain_zome_types =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-holochain_keystore =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-holochain =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-hdk =  { branch = "re-expose-useful-apis", git = "https://github.com/holochain/holochain.git"}
-# holochain_p2p = "0.0.68"
-# holochain_types = "0.0.69"
-# holochain_zome_types = "0.0.58"
-# holochain_keystore = "0.0.67"
-# holochain = { version = "0.0.175" }
-# hdk = "0.0.163"
+holochain_util = "0.1.0-beta-rc.0"
+holochain_p2p =  "0.1.0-beta-rc.0"
+holochain_types = "0.1.0-beta-rc.0"
+holochain_zome_types =  "0.1.0-beta-rc.0"
+holochain_keystore =  "0.1.0-beta-rc.0"
+holochain =  "0.1.0-beta-rc.0"
```

Comment:
Before you publish, can you bump this to rc.1? I they fixed an important bug. Only holochain and holochain cli were bumped I think